### DISCUSSION TITLE
How to build dependencies for Apple TVos?

### DISCUSSION THREAD

**Discussion by foreverdebugging** [microsoft/vcpkg#39839] [General]
Vcpkg doesn't support the Apple tvos platform out of the box? What do I need to do to add support?

**Comment by autoantwort** [comment]


**Reply by autoantwort** [reply]
Change `VCPKG_CMAKE_SYSTEM_NAME` to what you need in the example

**Reply by foreverdebugging** [reply]
I tried this, unfortunately there is no TVOS value for VCPKG_CMAKE_SYSTEM_NAME.

**Reply by autoantwort** [reply]
According to https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_NAME.html#system-names-known-to-cmake the value is `tvOS`

**Reply by foreverdebugging** [reply]
No it's a bit more complicated than that, I've already been answered in another issue. It would require changes to vcpkg. https://github.com/microsoft/vcpkg/issues/39842

**Reply by autoantwort** [reply]
You can also provide your own toolchain and set `VCPKG_CHAINLOAD_TOOLCHAIN_FILE` in the triplet
